packages/snap/src/stateManagement.ts

```diff
@@ -58,15 +61,15 @@ const defaultState: KeyringState = {
 export async function getState(): Promise<KeyringState> {
   const state = (await snap.request({
     method: 'snap_manageState',
-    params: { operation: 'get' },
+    params: { operation: ManageStateOperation.GetState },
   })) as any;
 
-  logger.debug('Retrieved state:', JSON.stringify(state));
+  if (!state) {
+    return defaultState;
+  }
 
-  return {
-    ...defaultState,
-    ...state,
-  };
+  logger.debug('Retrieved state:', JSON.stringify(state));
```

Comment:
Do we want to keep this one? Might be useful, but IDK if this was intended?

Reply:
Also, dumping the state might leak potential secrets, but that's maybe fine since it's more or less a "development" Snap, right?